src/Akka.Persistence.PostgreSql/Journal/PostgreSqlQueryExecutor.cs

```diff
@@ -264,7 +266,8 @@ public class PostgreSqlQueryConfiguration : QueryConfiguration
             string defaultSerializer, 
             JsonSerializerSettings jsonSerializerSettings = null, 
             bool useSequentialAccess = true, 
-            bool useBigIntPrimaryKey = false)
+            bool useBigIntPrimaryKey = false,
+            int tagsColumnSize = 2000)
```

Comment:
Technically, introducing this argument is a binary breaking change on the constructor signature - but I don't think there are any libraries out there that use `PostgreSqlQueryConfiguration` so it's probably minor.